tests/data/spe1_st/tests/test_webviz_ert.py

```diff
@@ -55,20 +55,21 @@ def test_webviz_response_correlation(dash_duo):
     )
 
     _verify_keys_in_menu(
-        dash_duo, plugin, response_keys, "parameter-selector-multi-resp"
+        dash_duo, plugin, ["WOPT:PROD"], "parameter-selector-multi-resp"
```

Comment:
i suggest making a new variable, `responses_with_observations`, to make it clearer why this test needed touching up